app/models/like.rb

```diff
@@ -7,6 +7,6 @@ class Like < ApplicationRecord
   private
 
   def update_counter
-    post.increment!(like_counter)
+    post.increment!(likes_counter)
```

Comment:
- [ ] This is a suggestion on how to fix the issue.😊

```suggestion
    post.increment!(:likes_counter)
```